### ISSUE TITLE
run as User without password

### ISSUE THREAD

**Issue by r9guy** [TWC-Software/DaemonMaster#15]
Then checking checkbox to run service as specefic user,  if selected user doesnt have password DaemonMaster doesnt proceed and insist user password to be inserter. This way program unable to create services for user accounts with blank passwords.

**Comment by MCPC10** [comment]
Due to security risks I doesn't implement such an feature for now.
Here is a link to such a topic:
https://stackoverflow.com/questions/1047854/cant-run-a-service-under-an-account-which-has-no-password

**Comment by MCPC10** [comment]
But I can allow to use empty passwords and than you must manually disable this security feature.

**Comment by r9guy** [comment]
I read the article @the link. I am aware of this security policy and at work i am keeping my computers with group policy to allow use of blank password. Because i usually use RDP to connect to industrial PCs. But i think the issue does  ot apply here.
Correct me if im wrong . Regardless group polocy settings 
1st.will Windows API permit service creation for localuser? If yes then there must be no security hole issue as this is windows DNA. 
2nd. If windows rill refuse then API funtiin must return error and functionality will be reliant exclusivly on users decision to have GP Enabled or Not, its not DeamonMaster who overcomes security right?

**Comment by MCPC10** [comment]
Sorry for the late answer.
So what you want is, that I should not check that the password is empty so that windows can decide this?

**Comment by r9guy** [comment]
Yes. If thats part of windows behavior.

**Comment by MCPC10** [comment]
Ok I will change it.

**Comment by MCPC10** [comment]
So I changed the code so that windows check password/right to start without password.
(It will be in the next release version of DM. xD)

**Comment by r9guy** [comment]
I checked the changes. while on local account it worked for me without password , with the accounts which do have password I have got an error message then I tried to run the service after creation. 
so DaemonMaster  on service creation page , create with account, of Course I typed in Username, and intentionally omitted the password. DeamonMaster created the service but It couldn't start it with message  "service didn't start due to login failure".
so I fought of the scenario that a service can indeed be created without password if user doesn't have one, but password can be changed later, then the service will not be able to run . but lets face some facts, Windows services out there are running regardless of such password change events. so how shall it be?
one solution I do propose is to check the valid password then entering it and give a message that such password is wrong, or even for account without password gray-scale the password field and make it invalid.
